entgql/schema.go

```diff
@@ -110,6 +116,10 @@ func (e *schemaGenerator) prepareSchema() (*ast.Schema, error) {
 
 func (e *schemaGenerator) buildTypes() (map[string]*ast.Definition, error) {
 	types := make(map[string]*ast.Definition)
+	var defaultImplementedInterfaces []string
+	if e.relaySpec {
+		defaultImplementedInterfaces = append(defaultImplementedInterfaces, "Node")
+	}
```

Comment:
The previous code was better imo. Don't think this extra variable is needed, but if you still want to use it, give it a shorter name like `defaultIfaces`.